etc/linters.json

```diff
@@ -89,12 +90,13 @@
     },
     {
         "name": "textlint",
-        "args": "-c /etc/opt/goat/.textlintrc \"/goat/**\"",
+        "args": "-c /etc/opt/goat/.textlintrc --dry-run",
```

Comment:
Why dry run?

Reply:
Dug back into the linter docs. Yep, unnecessary. Removed.